ansible/roles/girder/tasks/main.yml

```diff
@@ -48,14 +48,9 @@
   sudo: yes
   tags: girder
 
-- name: probe hpccloud folder
-  stat: path=/opt/hpccloud/girder
-  register: path
-
 - name: Get Girder from github
   git: repo=https://github.com/girder/girder.git version={{ girder_version }} dest=/opt/hpccloud/girder force=yes accept_hostkey=yes
   sudo: yes
-  when: path.stat.exists == False
```

Comment:
We do a development install with pip from the /opt/hpccloud/girder directory.  this is similar to a ```python setup.py develop``` but will ensure that all the requirements are installed as well.